contrib/database/sql/option.go

```diff
@@ -31,6 +31,7 @@ type config struct {
 	tags               map[string]interface{}
 	dbmPropagationMode tracer.DBMPropagationMode
 	dbStats            bool
+	statsTags		   []string
```

Comment:
Beware of formatting. Make sure to run `go fmt` and commit the changes.

Reply:
This should probably be a part of your pipeline if everyone is required to run the same auto-formatting.

Reply:
As detailed in the [style guide](https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines) linked in our [contribution guide](https://github.com/DataDog/dd-trace-go/blob/main/CONTRIBUTING.md#style-guidelines), we follow the official [Go Code Review Comments guide](https://go.dev/wiki/CodeReviewComments#gofmt):

> Run [gofmt](https://pkg.go.dev/cmd/gofmt/) on your code to automatically fix the majority of mechanical style issues. Almost all Go code in the wild uses gofmt. The rest of this document addresses non-mechanical style points.

We encourage PRs to be in proper shape before merging, with no pipelines automagically updating your code.